cmake/install_mujoco.cmake

```diff
@@ -0,0 +1,31 @@
+
+
+macro(INSTALL_MUJOCO)
+
+  # installing if not installed
+  # todo: should message with STATUS mode, but then
+  # nothing shows up in the terminal. Using INFO for now
+  # (but print to stderr)
+  set(local_mujoco "$ENV{HOME}/.mpi-is/mujoco")
+  message(INFO " installing mujoco in ${local_mujoco}")
+  if (NOT EXISTS ${local_mujoco})
```

Comment:
Shouldn't there also be a check for `/opt/mpi-is/mujoco` here?

Reply:
I added a script "install_folder" which print the install folder (depending if called with sudo rights or not), and then check the existence of the folder printed. The same "install_folder" script is then called by "install" in order to install in the correct folder.